From 55354caf6c055d365d7cc90ab8e2727ba99300ca Mon Sep 17 00:00:00 2001
From: Ben Dooks <ben.dooks@codethink.co.uk>
Date: Wed, 18 Dec 2024 10:39:23 +0000
Subject: [PATCH 21/21] temp: allow bpf test built-in

---
 lib/Kconfig.debug | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index dc0e0c6ed075..7fa9acda9945 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -2588,7 +2588,7 @@ config TEST_VMALLOC
 
 config TEST_BPF
 	tristate "Test BPF filter functionality"
-	depends on m && NET
+	depends on NET
 	help
 	  This builds the "test_bpf" module that runs various test vectors
 	  against the BPF interpreter or BPF JIT compiler depending on the
-- 
2.51.0.windows.1

